.dotnet/src/Properties/AssemblyInfo.cs

```diff
@@ -3,4 +3,4 @@
 
 using System.Runtime.CompilerServices;
 
-// [assembly: InternalsVisibleTo("OpenAI.Tests")]
+// [assembly: InternalsVisibleTo("OpenAI.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b197326f2e5bfe2e2a49eb2a05bee871c55cc894325b3775159732ad816c4f304916e7f154295486f8ccabefa3c19b059d51cd19987cc2d31a3195d6203ad0948662f51cc61cc3eb535fc852dfe5159318c734b163f7d1387f1112e1ffe10f83aae7b809c4e36cf2025da5d1aed6b67e1556883d8778eeb63131c029555166de")]
```

Comment:
Not super important, but this isn't needed if we don't plan on signing the tests project.